### PR TITLE
Update small items on the website

### PR DESCRIPTION
**Pull request by abachma2**
This PR addresses some of the items in #337, such as missing `>` in the tutorial, fixing links to places, updating projects that use Cyclus, listing funding sources for Cyclus, etc. Others came up with some of the items this PR addresses, so definitely welcoming feedback on some of the changes.

### CODE REVIEW

File: source/index.rst [hunk 1]
```diff
@@ -73,6 +51,26 @@ Learn More
     previous/index
     cite/index
 
+
+News
+----
+
```
Comment (gonuke) [inline source/index.rst:58]: Should we generate some news? Maybe @abachma2's PhD?

File: source/user/install.rst [hunk 2]
```diff
@@ -40,21 +40,16 @@ will identify the best approach for you.
...
-    install_from_tarball
 
-
-2. Run Cyclus with a Sample XML File
+1. Run Cyclus with a Sample XML File
 -------------------------------------
 
-Try running |Cyclus| for yourself. The result will be a :doc:`database <dbdoc>` named cyclus.sqlite.  Use the drop down menu to load the sqlite file into Cyclist for data visualization, or use your favorite sqlite browser to peruse.
+Try running |Cyclus| for yourself. The result will be a :doc:`database <dbdoc>` named cyclus.sqlite.  Use the drop down menu to 
+load the sqlite file into Cyclist for data visualization, or use your favorite sqlite browser to peruse.
```
Comment (gonuke) [inline source/user/install.rst]: Shouldn't we remove reference to `Cyclist` here?

**Review by gonuke — COMMENTED** [review]
A couple of little suggestions - thanks @abachma2

**Review by gonuke — CHANGES_REQUESTED** [review]
I think this needs an update/rebase, too

File: source/arche/tutorial_py/setup.rst [hunk 3]
```diff
@@ -70,7 +70,7 @@ Now we can install the tutorial project via,
 .. code-block:: console
 
     ~ $ cd tutorial
-    ~/tutorial $ python setup.py install --user
+    ~/tutorial $ pip install .
```
Comment (bennibbelink) [inline source/arche/tutorial_py/setup.rst]:
```suggestion
    ~/tutorial $ python -m pip install .
```

File: source/arche/tutorial_py/setup.rst [hunk 4]
```diff
@@ -70,7 +70,7 @@ Now we can install the tutorial project via,
...
 
 
 Let's now make an example input file in a special ``input`` directory:
```
Comment (bennibbelink) [inline source/arche/tutorial_py/setup.rst:75]: GitHub isn't letting me add a comment on line 84 but I think it's missing a `py` after `input/storage.`

**Review by bennibbelink — CHANGES_REQUESTED** [review]
Suggested a couple minor changes.  I have one additional question about the example `setup.py` in this tutorial - why is `'xo'` included as a script?  Is it just included as an indication that users can include their own scripts in the package or is actually necessary for this tutorial?  I removed this dependency from the Dockerfile in [#1735](https://github.com/cyclus/cyclus/pull/1735) and want to make sure I'm not breaking anything.

File: source/index.rst (hunk 1 above)
Comment (abachma2) [reply source/index.rst:58]: @nuclearkatie had an item about the latest news being from 2018, but I wasn't sure what news items we could add. We could add an announcement about the current NEUP grant, show that there is active development on the code.

**Comment by abachma2** [comment]
> I think this needs an update/rebase, too

Do we do rebase, or merge with this repo? I think rebase.

**Comment by abachma2** [comment]
> Suggested a couple minor changes. I have one additional question about the example `setup.py` in this tutorial - why is `'xo'` included as a script? Is it just included as an indication that users can include their own scripts in the package or is actually necessary for this tutorial? I removed this dependency from the Dockerfile in [#1735](https://github.com/cyclus/cyclus/pull/1735) and want to make sure I'm not breaking anything.

I don't know about that one. @gonuke might have some more insight on what that does for the build.

File: source/index.rst [hunk 5]
```diff
@@ -30,32 +30,10 @@ The |Cyclus| project repository is located at http://github.com/cyclus/cyclus
...
-**June 10, 2015:** First Cyclus :doc:`user tutorial <user/tutorial/index>` and :doc:`developer tutorial <arche/tutorial/index>` are held at the 2015 ANS Annual Meeting in San Antonio, TX
-
-**June 7, 2015:** :doc:`Cyclus v1.3.1 is released <previous/v1.3>`
 
 .. toctree::
     :maxdepth: 1
-
-    news/fy16neup
-    newsletters/oct2017
+    user/install
```
Comment (bennibbelink) [inline source/index.rst:37]: I'm not seeing the `toctree` under Quick Start.  This screenshot is from the build artifact generated by CI:

Screenshot 2024-04-15 at 9 45 54 AM

File: source/user/install.rst [hunk 6]
```diff
@@ -40,21 +40,17 @@ will identify the best approach for you.
...
 
-    install_binary
-    install_docker
-    install_custom
-    install_from_git
-    install_from_tarball
 
-
-2. Run Cyclus with a Sample XML File
+1. Run Cyclus with a Sample XML File
```
Comment (bennibbelink) [inline source/user/install.rst]:
```suggestion
2. Run Cyclus with a Sample XML File
```

File: source/index.rst [hunk 7]
```diff
@@ -73,6 +51,26 @@ Learn More
     previous/index
     cite/index
 
+
+News
+----
+
+**June 1st, 2018:** :doc:` <newsletters/may2018>`
```
Comment (bennibbelink) [inline source/index.rst]: Add text for this `:doc:` reference.  Currently rendering incorrectly: 
```suggestion
**June 1st, 2018:** :doc:`Cyclus Newsletter Spring 2018! <newsletters/may2018>`
```
Screenshot 2024-04-15 at 9 49 21 AM

**Review by bennibbelink — CHANGES_REQUESTED** [review]

**Comment by abachma2** [comment]
Good catches @bennibbelink. Hopefully it all renders correctly now.

**Review by bennibbelink — APPROVED** [review]
Thanks @abachma2!

**Comment by gonuke** [comment]
> > Suggested a couple minor changes. I have one additional question about the example `setup.py` in this tutorial - why is `'xo'` included as a script? Is it just included as an indication that users can include their own scripts in the package or is actually necessary for this tutorial? I removed this dependency from the Dockerfile in [#1735](https://github.com/cyclus/cyclus/pull/1735) and want to make sure I'm not breaking anything.
> 
> I don't know about that one. @gonuke might have some more insight on what that does for the build.

`xo` is a lightweight python-based editor written by Scopatz - I think we can probably get rid of it and it's dependency.

**Review by bennibbelink — CHANGES_REQUESTED** [review]
In that case @abachma2 are you opposed to removing `'xo'` from line 63 of `tutorial_py/setup.rst`?

**Comment by abachma2** [comment]
I am always down for removing unneeded dependencies from installation instructions.

**Review by bennibbelink — APPROVED** [review]
Looks good.  Are we waiting to include some news items in this PR or should we merge and save that for another PR?

**Comment by abachma2** [comment]
I don't think that's a blocking thing for this PR. I think we have enough other items in #337 that can include a new News item in a later PR. Maybe we can discuss somewhere else what a good news item would be.

**Comment by bennibbelink** [comment]
I took the liberty of rebasing this PR.  @gonuke does that resolve your requested changes?

**Review by gonuke — APPROVED** [review]

**Pull request merged**